java/src/apps/gui3/TabbedPreferences.java

```diff
@@ -58,9 +58,8 @@ public String getHelpTarget() {
     }
 
     @Override
-    public String getTitle() {
-        return rb.getString("TitlePreferences");
-    }
+    public String getTitle() { return Bundle.getMessage("TitlePreferences"); }
```

Comment:
This needs to be multi line. We want to avoid making unnecessarily compact code.